src/uwu/getting-started.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Getting started
+lang: en-US
+sidebar: false
+---
+
+# Getting started
+
+## Installation
+
+::: tip Video tutorial
+ If you prefer using a video tutorial, Artesians made a beautiful one. You will find it... Oh oh, forgot we took it down 😭: we are really concerned by our users privacy.
+:::
+
+::: warning Warning
+  The installation is really difficult! Luckily we have a beautiful website which is fully PWA compliant. Forget about the app, you just need to press the "Install the app" notification on your phone. 
+:::
+
+### AltStore instructions for Mac
+1. Download **`Paperback.ipa`**
+1. Try to open it on your phone, that won't work.
+1. Just ask for help in #support
+
+## FAQ
+
+### How to install it on Android?
+Will the app work on Android? Maybe, ping Paper on Discord to check.
+
+### App crashes on launch
+It's a **bug**! Just go to #feedback channel on Discord and ask:
+> "paperback keeps on crashing when i open it, what should i do??"
+
+### What type of business model will you take on?
+There are no plans to adopt a formal business model but Paper is thinking of buying a new car thus it would be nice to get support for development of the app.
+
+### Why isn't the app on TestFlight?
+Apple decided to prevent Paper from getting an Apple Developper Account. That mean there must be something tricky about him, use the app at your own risk!
+
+### I can't find the fancy page reader
+It's normal: Paper didn't like it so he decided to remove it.
+
+### Does the app support multiple source?
+Of course, a good manga app need to support multiples sources. Just note that using them will mess up your library and make the app unusable.
+
+### Why should I join the your Discord?
+Our **[Discord](https://discord.gg/Ny83JV3)** is an active place where you will find weekly game events. Minecraft, Dota, League, we have everything to spend time waiting for Paper to make the app usable. We also provide real estate consulting and clothing suggestions.
+
+### I have some other issue that isn't listed here
+It's not possible, everything had been listed here. The #support channel is not designed to help people who did not read this guide.
```

Comment:
```suggestion
That's absolutely impossible, everything has been listed here. The #support channel is not designed to help people who did not read this guide.
```

Honestly, I don't think the second half of this section is very nice. My personal suggestion would just be..
```suggestion
That's literally impossible.
```
Just leaving it at this.

Reply:
I used the last one. Thanks